rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -155,4 +157,23 @@ private static void signLegacyTransactionInputFromP2shMultiSig(BtcTransaction tr
             input.setScriptSig(inputScriptSig);
         }
     }
+
+    public static List<BtcECKey> getDefaultPublicKeys() {
```

Comment:
What's the use for this? Seems to be very specific

Reply:
This method gets the default public keys. It'll be use only for tests.

Reply:
Done! Started using the `getBtcPublicKeys` method from the active federation. Reversed the changes on `BitcoinTestUtils.`